Dockerfile

```diff
@@ -1,5 +1,6 @@
 ### STAGE 1: Build ###
-# This image is around 50 megabytes
+# This image is around 53 megabytes
+# https://hub.docker.com/_/node/tags?page=1&name=alpine
```

Comment:
add this part pls 
https://github.com/Visual-Regression-Tracker/frontend/pull/332/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R9-R14

it solves this issue 
https://sonarcloud.io/project/security_hotspots?id=Visual-Regression-Tracker_frontend&hotspots=AYn0nEVqSdwZZQcpatz_